Client/Frontend/Browser/Tab.swift

```diff
@@ -151,7 +150,7 @@ class Tab: NSObject {
 
     init(configuration: WKWebViewConfiguration, type: TabType = .regular) {
         self.configuration = configuration
-        rewardsId = UInt32.random(in: UInt32.min...UInt32.max)
+        rewardsId = UInt32.random(in: 1...UInt32.max)
```

Comment:
I feel that we should make a unit test and verify that a new tab will always have rewardsId >0 to catch regressions. I know it's a small thing, but we have to start somewhere

Reply:
Im not entirely sure thats doable because its a random number generating between 1 and 4294967295, the test would pass 99.99999% of the, wouldn't it?

Reply:
Yeah I guess, and it's a let constant not a var so you are right